ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -91,7 +91,9 @@ function suite() {
         TESTS="$(ssh "${SSHOPTS[@]}" "root@${IP}" openshift-tests run --dry-run --provider "${TEST_PROVIDER}" "${TEST_SUITE}")"
         echo "${TESTS}" | grep -v "${TEST_SKIPS}" >/tmp/tests
         echo "Skipping tests:"
+        set +e
```

Comment:
Since our goal for this step it's to keep it as much as possible aligned with the `ci-operator/step-registry/openshift/e2e/test` one (so that we could move to adopt it directly), I think it'd be worth checking its current status (see https://github.com/openshift/release/blob/30d7dab48309c88d59722fe5667e93a9a6b338c1/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L233-L250) and update our code respectively. Looks like they already addressed this specific issue.

Reply:
Thank you @andfasano, good point. I'll change that way.

Reply:
I think still similar problem is also in here https://github.com/openshift/release/blob/30d7dab48309c88d59722fe5667e93a9a6b338c1/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L233-L250.

There are chains and it returns exit_code with 1, when TEST_SKIPS is empty. Thereby, `TEST_ARGS="${TEST_ARGS:-} --file /tmp/tests" ` is never set. 

With or without chains, we should catch error code and return 0 to pass through https://github.com/openshift/release/blob/30d7dab48309c88d59722fe5667e93a9a6b338c1/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L240